SortVisualization/SortVisualization/Models/TypeSortEntity.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  TypeSortEntity.swift
+//  SortVisualization
+//
+//  Created by Oleg Yankiwskyi on 5/7/18.
+//  Copyright © 2018 Oleg Yankiwskyi. All rights reserved.
+//
+
+import Foundation
+
+class TypeSortEntity {
```

Comment:
`class` may easily replaced with `struct`. `init` may be removed, since default one suits you